.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: github-repo-stats
+# See https://github.com/jgehrcke/github-repo-stats
+
+# Controls when the workflow will run
+on:
+  schedule:
+    # Run this once per day.
+    # The specific hour is interpreted in UTC time zone.
+    # Do this towards the end of the UTC day.
+    - cron: "0 23 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+jobs:
+  j1:
+    name: github-repo-stats
+    runs-on: ubuntu-latest
+    steps:
+      - name: GHRS
+        uses: jgehrcke/github-repo-stats@HEAD
+        with:
+          ghpagesprefix: https://jgehrcke.github.io/ghrs-test
```

Comment:
Ah. now I see the source of your questions on slack.
Yes. something doesn't smell good here the way this looks.
Don't know enough to put my finger on it.